src/components/molecules/EarthEditor/DatasetPane/DatasetModal/Gdrive/index.tsx

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { styled, useTheme } from "@reearth/theme";
+
+import Icon from "@reearth/components/atoms/Icon";
+import Flex from "@reearth/components/atoms/Flex";
+import Loading from "@reearth/components/atoms/Loading";
+import Divider from "@reearth/components/atoms/Divider";
+import Text from "@reearth/components/atoms/Text";
+import { metricsSizes } from "@reearth/theme/metrics";
+import AssetCard from "@reearth/components/molecules/Common/AssetModal/AssetCard";
+import AssetListItem from "@reearth/components/molecules/Common/AssetModal/AssetListItem";
+import Button from "@reearth/components/atoms/Button";
+import useHooks, { GoolgeSheet, SheetParameter } from "./hooks";
+
+export type Props = {
+  onReturn: () => void;
+  onSheetSelect: (sheet: SheetParameter) => void;
+  syncLoading: boolean;
+};
+
+const Gdrive: React.FC<Props> = ({ onReturn, syncLoading, onSheetSelect }) => {
+  const intl = useIntl();
+  const theme = useTheme();
+
+  const {
+    isLoading,
+    pickedFile,
+    pickedFileSheets,
+    selectedSheet,
+    handleAuthClick,
+    handleSheetSelect,
+  } = useHooks(onSheetSelect);
+
+  return (
+    <>
+      <StyledIcon icon={"arrowLongLeft"} size={24} onClick={onReturn} />
+      {syncLoading && <Loading />}
+      {pickedFile?.id && (
```

Comment:
Can you move the Flex and Divider outside of both conditional areas? Both areas (whether pickedFile?.id or !pickedFile?.id) include the first Flex container and its contents + Divider so I don't see why they can't be moved outside to always be shown.